src/pointer.cr

```diff
@@ -495,10 +499,22 @@ struct Pointer(T)
   # ptr.to_slice(6) # => Slice[0, 0, 0, 13, 14, 15]
   # ```
   def clear(count = 1)
-    Intrinsics.memset(self.as(Void*), 0_u8, (count * sizeof(T)).to_u32, 0_u32, false)
+    Intrinsics.memset(self.as(Void*), 0_u8, bytesize(count), 0_u32, false)
   end
 
   def clone
     self
   end
+
+  private def bytesize(count)
+    {% if flag?(:bits64) %}
+      count.to_u64 * sizeof(T)
+    {% else %}
+      if count > UInt32::MAX
+        raise ArgumentError.new("Negative count")
```

Comment:
One moar copy/pasted exception message to fix.

Reply:
Fixed!